### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by Goose-Of-War**
Updating the README. Someone review

### CODE REVIEW

**Comment by Goose-Of-War** [comment]
@ItsAnkan @SachdevJai have a look at the README and tell me if I need to add something more

**Review by SachdevJai — CHANGES_REQUESTED** [review]
grammar in line 2 i think it should be at instead of in 
and the 2 line express router thing since we're switching to one liners anyway (maybe this should be in readme like this)

**Comment by SachdevJai** [comment]
other than that looks good to me

**Comment by Goose-Of-War** [comment]
> grammar in line 2 i think it should be at instead of in 
> and the 2 line express router thing since we're switching to one liners anyway (maybe this should be in readme like this)

So do you think I should wait for revamp router, edit and then merge this?

File: README.md
Comment (ItsAnkan) [inline README.md:1]: if you changed the md codeblock language to jinja for default page, why not change if for newsletter also ?

Comment (Goose-Of-War) [reply README.md:1]: Forgot ;-;

File: README.md [hunk 1]
```diff
@@ -117,19 +103,90 @@ The default newsletter template is:
...
+
+- For any new route, make sure to use a new router file instead of adding to `route.js`
+- Make sure to extend your template from either `_base.njk` or another template (in case of similar pages, like events or newsletters).
+- Make sure the code passes the lint and mocha tests. You can check that by running `npm run test`. The ESLint configuration can be seen in `.eslintrc.json` and tests can be seen in the `test` folder.
+- If you have any questions, post 'em in the WebDev channel.
+- If you aren't a member of the society but have queries/reports/suggestions, feel free to use the [Issues](https://github.com/kgpmask/MASK/issues)/[Discussions](https://github.com/kgpmask/MASK/discussions) pages.
+
+---
+
 ## Credits:
```
Comment (ItsAnkan) [inline README.md:168]: i can't find the official source, but there should be no trailing punctuation after headings, or so as my linter says
keep them as is ig

**Review by ItsAnkan — APPROVED** [review]
Looks good ig, there are many warnings given by my MD linter, but I can't find any sources to backup those warning claims, so.. is good ig 👍

File: README.md (hunk 1 above)
Comment (Goose-Of-War) [reply README.md:168]: But where heading?

Comment (Goose-Of-War) [reply README.md:180]: Guidelines had punctuations?

Comment (ItsAnkan) [reply README.md:180]: "Credits"

File: README.md [hunk 2]
```diff
@@ -117,19 +103,102 @@ The default newsletter template is:
...
+
+module.exports = {
+	route: '/path'
+	router;
+}
+```
+
+All routers are imported and used in `route.js` in the `src` folder. 
+
+```js
```
Comment (ItsAnkan) [inline README.md:148]: why 1 tab more in each line, this looks shifted by one tab tbh

Comment (Goose-Of-War) [reply README.md:148]: Because the actual code was ;-;
Should I shift it back?

Comment (ItsAnkan) [reply README.md:148]: ig you should, cuz you are displaying just the code where the "thing" is happening

Comment (ItsAnkan) [reply README.md:148]: nice

**Review by ItsAnkan — APPROVED** [review]
Looks good, im gonna have mercy on you by not stating each and every error given by my linter

**Review by SachdevJai — APPROVED** [review]
seems good

**Pull request merged**